src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiSelect/MultiSelect.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react';
+import type {Element} from 'react';
+import type {SelectChildren} from '../Select/types';
+import {GenericSelect, Option} from '../Select';
+
+type Props = {
+    values: Array<string>,
+    label: string,
+    onChange: (values: Array<string>) => void,
+    children: SelectChildren,
+    icon?: string,
+};
+
+export default class MultiSelect extends React.PureComponent<Props> {
+    static defaultProps = {
+        values: [],
+    };
+
+    optionIsSelected = (option: Element<typeof Option>): boolean => {
+        return this.props.values.indexOf(option.props.value) !== -1;
+    };
+
+    handleSelect = (value: string) => {
+        let newValues = [...this.props.values];
+        const index = newValues.indexOf(value);
+        if (index === -1) {
+            newValues.push(value);
+        } else {
+            newValues.splice(index, 1);
+        }
+        this.props.onChange(newValues);
+    };
+
+    getLabelText = () => this.props.label;
+
+    render() {
+        return (
+            <GenericSelect
```

Comment:
At first i wanted it to call `AbstractSelect` because it should just be used by other components which further specify the select behavior. I fear that if we just call it `Select` users (developers) will be confused as it has the same name as the html component but shouldn't be used directly.